qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -86,23 +86,28 @@ if [ `id -u` -ne 0 ]
 then
     expect_false rbd-nbd map ${IMAGE}
 fi
+
 expect_false _sudo rbd-nbd map INVALIDIMAGE
 expect_false _sudo rbd-nbd --device INVALIDDEV map ${IMAGE}
 
 # map test using the first unused device
 DEV=`_sudo rbd-nbd map ${POOL}/${IMAGE}`
-_sudo rbd-nbd list-mapped | grep "^${DEV}$"
+_sudo rbd-nbd list-mapped | grep "${DEV} $"
+_sudo rbd-nbd list-mapped | grep "${POOL}"
+_sudo rbd-nbd list-mapped | grep "${IMAGE}"
```

Comment:
Theoretically, it might give false positive, e.g. if "$POOL" is matched for some other mapped image. I would prefer if you test for all values simultaneously. Additionally you could extract `pid`:
```
PID=`_sudo rbd-nbd list-mapped | awk '$2 == '${DEV}' && $3 == '${POOL}' && $4 == '${IMAGE}' {print $1}'`
test -n "${PID}"
```
Then you could use `PID` to check it matches rbd-nbd process:
```
ps -p ${PID} -o cmd | grep rbd-nbd
```
and also add a test that would kill `rbd-nbd` process by PID and test the process has died and the device is properly cleaned up.